www/js/metrics-mappings.js

```diff
@@ -389,7 +389,7 @@ angular.module('emission.main.metrics.mappings', ['emission.plugin.logger',
       }
     }
 
-    $ionicPlatform.ready().then(function() {
+    $ionicPlatform.ready().then(() => {
       DynamicConfig.configReady().then((newConfig) =>
         this.init()
       );
```

Comment:
again, I don't see how this fixes anything. `this` is always the window, which does not have an `init` function.
See https://github.com/e-mission/e-mission-phone/pull/938

Reply:
Actually, when I was fixing the dashboard bugs myself it did fix the issue. However, with your changes this is probably irrelevant

(In traditional JS functions, `this` is always the window. But in arrow functions, `this` is inherited from the parent scope.)

Reply:
reverted 75e9468f90af16667ace1fb96df06fe8dc7e1bdb